lib/ret_web/views/api/v1/room_view.ex

```diff
@@ -0,0 +1,82 @@
+defmodule RetWeb.Api.V1.RoomView do
+  use RetWeb, :view
+  alias Ret.{Hub, Scene, SceneListing}
+
+  # def render("create.json", %{hub: hub}) do
+  #   %{
+  #     status: :ok,
+  #     hub_id: hub.hub_sid,
+  #     url: hub |> Hub.url_for(),
+  #     creator_assignment_token: hub.creator_assignment_token,
+  #     embed_token: hub.embed_token
+  #   }
+  # end
+
+  def render("show.json", %{hub: %Hub{scene: %Scene{}} = hub, embeddable: embeddable}) do
+    hub |> render_with_scene(embeddable)
+  end
+
+  def render("show.json", %{hub: %Hub{scene_listing: %SceneListing{}} = hub, embeddable: embeddable}) do
+    hub |> render_with_scene(embeddable)
+  end
+
+  def render("show.json", %{hub: hub}) do
+    hub |> render_with_scene_asset(:gltf_bundle, hub.default_environment_gltf_bundle_url)
+  end
+
+  def render_with_scene(hub, embeddable) do
+    %{
+      # hub_id: hub.hub_sid, # renamed hub_id to room_id
+      room_id: hub.hub_sid,
```

Comment:
This doesn't need to block this PR necessarily, but we should talk about this `<type>_id` convention in API responses rather than just using `id`. IMO we should be using `id` when it is already explicit what type of record is being returned. Internally, in Reticulum code, I don't see a problem with it though. It helps me keep ids straight.